modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -60,97 +36,91 @@ output application/dw
-
-
 //com.mulesoft.dw.dumper_fill_stacktrace//
 |`com.mulesoft.dw.dumper_fill_stacktrace`|
 _Experimental:_ Adds the StackTrace for dumper exceptions when `com.mulesoft.dw.dump_files` is set to `true`.
-
 * Type: `Boolean`
 * Default: `false`
-
-`com.mulesoft.dw.dumper_fill_stacktrace` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave.
-
-
-//com.mulesoft.dw.error_value_length//
-|`com.mulesoft.dw.error_value_length`|
-Sets the maximum length of exception messages to display to the user. The message is truncated to the maximum length. This setting is useful for avoiding long exception messages.
-
-* Type: `Number`
-* Default: `80`
-
-
+`com.mulesoft.dw.dumper_fill_stacktrace` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave. See xref:dataweave-troubleshoot.adoc[Troubleshooting a Failing DataWeave Script].
 //com.mulesoft.dw.indexsize//
 |`com.mulesoft.dw.indexsize`|
 Sets the maximum size (in bytes) of the page in memory that indexed readers use.
-
-* Type: `Number`
-* Default: `1572864` (1.5 MB)
-
-
+* Type: `Integer`
+* Default: `1572864`
+//com.mulesoft.dw.java.stacktrace//
+|`com.mulesoft.dw.java.stacktrace`|
+Specifies the depth of the java stacktrace
+* Type: `Integer`
+* Default: `6`
 //com.mulesoft.dw.max_memory_allocation//
 |`com.mulesoft.dw.max_memory_allocation`|
-Sets the size (in bytes) of each slot in the off-heap memory pool.
-
-* Type: `Number` (in bytes)
-* Default: `1572864` (1.5 MB)
-
-See xref:dataweave-memory-management.adoc[DataWeave Memory Management].
-
-
+Sets the size (in bytes) of each slot in the off-heap memory pool. DataWeave stores payloads that exceed this size in temporary files `dw-buffer-input-${count}.tmp` and `dw-buffer-output-${count}.tmp`.
+* Type: `Integer`
+* Default: `1572864`
 //com.mulesoft.dw.memory_pool_size//
 |`com.mulesoft.dw.memory_pool_size`|
 Sets the number of slots in the memory pool. DataWeave buffers use off-heap memory from a pool, up to a defined size (`com.mulesoft.dw.memory_pool_size` * `com.mulesoft.dw.max_memory_allocation`). DataWeave allocates the remainder using heap memory. See xref:dataweave-memory-management.adoc[DataWeave Memory Management].
-
-* Type: `Number`
+* Type: `Integer`
 * Default: `60`
-
-
-//com.mulesoft.dw.multipart.defaultContentType//
-|`com.mulesoft.dw.multipart.defaultContentType`|
-Sets the default Content-Type to use on parts of the `multipart/*` format when a `Content-Type` is not specified. See also, the multipart reader property xref:dataweave-formats-multipart.adoc#properties[defaultContentType]. _Introduced in DataWeave 2.3 (2.3.0-20210720) for the August 2021 release of Mule 4.3.0-20210719._
-
-* Type: `String`
-* Default: `application/octet-stream`
-
-
+//com.mulesoft.dw.scheduler.size//
+|`com.mulesoft.dw.scheduler.size`|
+Specifies the size of DW scheduler thread pool
+* Type: `Integer`
```

Comment:
```suggestion

* Type: `Number`
```

Reply:
Is this the number of threads in the thread pool? Or some other unit? Need to say what that is.

Reply:
asked on channel

and Number or Integer, as elsewhere